bramkouwen/Music-Mixer
# Adding Music Icons

## What does the PR do?
* It places the music icons in the images folder


## How do you check this PR?
* Pull down the branch
*  See added image files

## Screenshots 
None.
